pkgs/tools/admin/pgadmin/default.nix

```diff
@@ -115,16 +61,8 @@ pythonPackages.buildPythonApplication rec {
     substituteInPlace docs/en_US/contributions.rst \
       --replace "code_snippets" ""
     patchShebangs .
```

Comment:
Would be nice if this could be scoped to a directory or a few

Reply:
Hi Sandro, I'm not sure what you mean?

Reply:
Replace `.` with specific folders if possible.

Reply:
done :+1: